CMakeLists.txt

```diff
@@ -251,7 +251,7 @@ if (BUILD_CGAME)
     FILES
         ${CGAMELIST}
     LIBS
-        RMLUI_LIB
+        ${RMLUI_LIBRARY}
```

Comment:
What's this {$RMLUI_LIBRARY} stuff? I don't like the extra indirection when a simple library name works well.

Reply:
Using a `<NAME>_LIBRARY` variable is usually what we do that with other libraries (with variant of `<NAME>_LIBRARIES` when more than one libraries).

Actually the capitalized name without being a variable looks like a typo as some other place in CMake can parse variable names without `${}`, here `RMLUI_LIB` looked like a variable name but was the file name. We don't do that nowhere else.

If we don't want to use a variable, then it better be named `srclibs-rmlui` but not `RMLUI_LIB`, the last one is very confusing.

Reply:
See my new commit for an alternative hardcoding `srclibs-rmlui` instead.